src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -463,11 +473,15 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 
 	protected onDidAddViews(added: IAddedViewDescriptorRef[]): ViewletPanel[] {
 		const addedViews = super.onDidAddViews(added);
-		this.progress(Promise.all(addedViews.map(addedView => {
-			(<ExtensionsListView>addedView).show(this.normalizedQuery()).then(model => {
-				this.alertSearchResult(model.length, addedView.id);
-			});
-		})));
+
+		this.searchPromise.cancel();
```

Comment:
newly added views shouldnt be cancelling search on existing views

Reply:
done